dna-frontend/src/app/app.component.ts

```diff
@@ -9,7 +9,7 @@ import { Component } from '@angular/core';
 import { RouterOutlet } from '@angular/router';
 import { TUI_DIALOG_CLOSES_ON_BACK } from '@taiga-ui/cdk';
 import { of } from 'rxjs';
-import { AuthComponent } from './core/components/auth/auth.component';
+import { AuthComponent } from './pages/auth/auth.component';
```

Comment:
This is probably an unnecessary import now? It is not used in app.component.html or this file so it can be removed.